source/manual/create-a-local-transaction.html.md

```diff
@@ -0,0 +1,236 @@
+
+`/data/vhost/local-links-manager/releases/YYYYMMDDHHMMSS/tmp/LGSL_CODE.csv`
+
+Generate the CSV file and exit out of the rails console.
+
+```ruby
+CSV.open(file, 'w') do |csv|
+  csv << ["name", "slug", "url"]
+  LocalAuthority.order(name: :asc).each do |la|
+    csv << [la.name, la.slug, nil]
+  end
+end
+```
+
+Download the CSV file just you just created to your local machine...
+
+```bash
+$ gds govuk connect scp-pull -e integration backend:MACHINE_NO REMOTE_FILE_PATH LOCAL_FILE_PATH
+```
+
+Hand this over to the service product owner.
+
+_The `name` is only present in the CSV file to assist the product owner complete the URLs as differences between the local authority `name` and `slug` can make it tricky to match the local authority correctly. It is not used when importing the links into LLM._
+
+## Add the service to Publisher
+
+Create a PR that adds the following to the bottom of [this](https://github.com/alphagov/publisher/blob/a656ea44d0e3a086b096ff2b2053290f1b9fb4d9/data/local_services.csv) file:
+
+| LGSL | Description | Providing Tier |
+| ---- | ---- | ---- |
+| [LGSL_CODE](#lgsl-code) | [SERVICE_DESCRIPTION](#seek-content-design-advice) | [PROVIDING_TIER](#providing-tier) |
+
+Once your PR is merged and deployed, run [this rake task](https://github.com/alphagov/publisher/blob/main/lib/tasks/local_transactions.rake) in Publisher...
+
+```bash
+bundle exec rails local_transactions:fetch_and_clean
+```
+
+## Make the service unavailable for a nation in Frontend
+
+__Note__: This is an optional step that can be done when the new service is live. However, this step must be complete before starting:
+
+* [Get details of or create a service](#get-details-of-or-create-a-service)
+
+If the service is unavailable in one or more devolved nation, create a PR to add the service to the [Frontend local transaction configuration file](https://github.com/alphagov/frontend/blob/master/lib/config/local_transaction_config.yml).
+
+This can be configured with or without `button_text` and `button_link`. [Seek content design advice](#seek-content-design-advice) with the content.
+
+For example:
+
+```yaml
+LGSL_CODE:
+  unavailable_in:
+    Scotland:
+      title: "This service is not available in Scotland"
+      body: "We do not know if they offer this service. Search the Scottish Government website to see if they do, or what else they offer."
+      button_text: "Find information for Scotland"
+      button_link: "https://www.mygov.scot"
+    Northern Ireland:
+      title: "This service is not available in Northern Ireland"
+      body: "This service is not available in Northern Ireland. You can find other services on your council website"
+```
+
+## Add the service to Local Links Manager
+
+__Note__: These steps must be complete before starting:
+
+* [Get details of or create a service](#get-details-of-or-create-a-service)
+* [Add the service to Publisher](#add-the-service-to-publisher)
```

Comment:
Yeah, it's not a very linear process unfortunately. You can do these in order, but it can waste a lot of time if you do that. I'll see if I can simplify this stuff.  Thanks

Reply:
I've removed these for now. Let's see how it goes in use.